src/react-apps/applications/runtime/src/components/ErrorBoundry.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+interface IErrorBoundry {
+  hasError: boolean;
+}
+
+class ErrorBoundry extends React.Component<any, IErrorBoundry> {
+  private static getDerivedStateFromError(error: Error) {
+    return {
+      hasError: true,
+    };
+  }
+
+  public state = {
+    hasError: false,
+  };
+
+  public componentDidCatch(error: Error, info: any) {
```

Comment:
Could info be something other than "any"?

Reply:
It returns an object, not sure if its any more descriptive, but changed :)